enableIPsec.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+oc  patch  network.operator.openshift.io/cluster --type=merge  -p  '{"spec": {"defaultNetwork" :{ "ovnKubernetesConfig" :{ "ipsecConfig" :{ "enable":true}}}}}'
```

Comment:
This should be in `hack` or deleted.

Reply:
All the content in the last commit will be deleted.

Reply:
In fact, I will just remove it now to save confusion. I thought it might someone with testing.